conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. It is important to be vulnerable in your group so that they understand your strengths and weaknesses within coding. Being open will help your team members know what you are capable of doing so they do not assign you something that you would not understand. This could prevent confusion and reserve time. Being open can also allow your team members to help you with problems with your code that you did not catch or understand.
+- Retaining team members ideas. This is a form of respecting everyone's thoughts and opinions. This means to keep or reserve someones idea to let your teammates know that they are heard within the group and their opinions matter.
+- Humbleness. Being humble is a form of humility it is important for everyone in a team to understand that it is not about one person but that it is about the team.
```

Comment:
The sentence structure here is a bit weird. Humbleness and humility are synonymous. So saying that "Being humble is a form of humility" is kinda stating the obvious. Instead, you could rephrase it to something along the lines of this:

```suggestion
- Encourage humility: Being humble is an important aspect of working in a team to. It helps in understanding that it is not about one person but that it is about the team. 
```
You can also expand that to talk about how it helps in avoiding conflict if you'd like.